pkgs/development/python-modules/pacup/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  poetry2nix,
+  python,
+  fetchFromGitHub,
+}:
+poetry2nix.mkPoetryApplication rec {
```

Comment:
Can we avoid using poetry2nix? It does handle security fixes rather bad.

Reply:
Ok, I replaced it with buildPythonPackage, specifying `format ="pyproject";`